tests/test_fft.py

```diff
@@ -69,15 +69,33 @@ def test_fft_ndim(self, dtype, shape, norm):
     @pytest.mark.parametrize("norm", [None, "forward", "ortho"])
     def test_fft_1D(self, dtype, n, norm):
         x = dpnp.linspace(-1, 1, 11, dtype=dtype)
-        a = dpnp.sin(x)
+        a = dpnp.sin(x)  # a.dtype is float16 if a.dtype is bool
         a_np = dpnp.asnumpy(a)
 
+        factor = 140 if dtype == dpnp.bool else 8
```

Comment:
Do we need to exclude `bool` dtype for that test if we have a dedicated test below?

Reply:
In this test when `dtype` is `bool`, `x.dtype` is bool but `a.dtype` would be `float16` which is not included by default. So, I guess it worth to keep it as it is. I added the additional test to cover `bool` dtype as well.